docs/django_search.rst

```diff
@@ -0,0 +1,122 @@
+Django Model Search Tool
+------------------------
+
+code_annotations django_find_annotations::
+    Usage: code_annotations django_find_annotations [OPTIONS]
+
+    Subcommand for dealing with annotations in Django models.
+
+    --config_file FILE                Path to the configuration file
+    --seed_safelist
+                                      Generate an initial safelist file based on
+                                      the current Django environment.  [default:
+                                      False]
+
+    --list_local_models
+                                      List all locally defined models (in the
+                                      current repo) that require annotations.
+                                      [default: False]
+
+    --report_path TEXT              Location to write the report
+    -v                              Verbosity level (-v through -vvv)
+    --lint                          Enable or disable linting checks  [default:
+                                      False]
+    --report                        Enable or disable writing the report
+                                      [default: False]
+    --coverage                      Enable or disable coverage checks  [default:
+                                      False]
+    --help                          Show this message and exit.
+
+
+Overview
+========
+The Django Model Search Tool, or Django Tool, is written to provide more structured searching and validation in a place
+where data is often stored. Since all of the models in a package can be enumerated it is possible, though not required,
+to use this tool to positively assert that **all** concrete (non-proxy, non-abstract) models in a project are annotated
+in some way. If you do not need this functionality and simply want to find annotations and create a report, the static
+search tool is much easier to configure and can search all of your code (instead of just model docstrings).
+
+.. important::
```

Comment:
This is a Sphinx admonition. In the processed docs it will highlight the text and put a border around it: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rst-directives